src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private Random random = new Random();
+    private ColorSupplier supplier = new ColorSupplier();
+
+    public Ball getRandomBall() {
+        Ball randomBall = new Ball();
+        randomBall.setColor(supplier.getRandomColor());
+        randomBall.setNumber(random.nextInt(100));

```

Comment:
lets move number 100 to constant https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names